integreat_cms/static/src/js/content-edit-lock.ts

```diff
@@ -28,7 +28,7 @@ const sendTakeOverMessage = async (url: string, payload: string) => {
 };
 
 const sendHeartbeat = async (heartbeatData: HTMLElement) => {
-    clearInterval(heartbeatInteval);
+    clearInterval(heartbeatInterval.ref.name);
```

Comment:
I don't really understand this change, why was it necessary?
With this line, I get the error:

> Uncaught (in promise) TypeError: heartbeatInterval is null

In the JS console.

Also, everywhere in the documentations about the `setInterval()`/`clearInterval()` functions it seems as we have to pass the return value of `setInterval()` directly to `clearInterval()` without any attributes:

```
intervalId = setInterval(function() { ... }, ...);
...
clearInterval(intervalId);
```

Reply:
I know and I am also very confused. But if you revert the change I get an error that this can't be assigned and webpack won't compile the js code :/

Reply:
I guess it's about the type of the variable.
Apparently, the return type was changed from `NodeJS.Timer` to `NodeJS.Timeout`...
We can prevent such errors in the future if we just use `ReturnType<typeof setTimeout>`, this should be platform independent...

Reply:
```suggestion
    clearInterval(heartbeatInterval);
```